### DISCUSSION TITLE
Vita3K compatible games crash on pc

### DISCUSSION THREAD

**Discussion by Olu196** [community/community#61098] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello Vita3k Devs. A job well done for all the work you've been doing on your PSVita emulator. I will gladly support you financially as I can. But 2 games on the playable list of Vita3k just don't run on my laptop. Mortal Kombat: Komplete Edition won't go beyond the intro screen and Ridge Racer won't get past the menu screen before crashing. Why is this? My laptop meets the recommended requirements of Vita3k. Please could you look into this. I hope games aren't being optimized mainly for android platform? This wouldn't be fair at all. B'cos this was originally an emulator for windows platform. Many thanks.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Comment by JaniVilen** [comment]
[Games](https://cricketagony.com/) compatible with the Vita3K emulator crashing on a PC can be due to various factors. It may result from compatibility issues, inadequate hardware specifications, outdated emulator versions, or configuration problems. Ensure your PC meets system requirements, use the latest emulator version, and consult emulator forums or documentation for troubleshooting tips.

**Comment by Olu196** [comment]
Good evening & thanks for replying. But why suggest that the fault could be
from my PC? I play other 3d games like Shinobido 2 & One Piece 3 on 3x
resolution, full speed. And let me mention that I noticed that the test
platform used for Mortal Kombat 9 was android. This is very misleading by
you because there are differences between android & PC. Yet PC users will
think that a game is playable. Alas! it was tested on android rather than
PC. How does that make sense if Vita3K was originally created for PC? It's
like you have abandoned the PC users of Vita3K for android. Maybe I'm the
only PC user complaining. But I don't think it matters. Others will notice
eventually. So please remove games like MK 9 & others tested only on
android from the 'playable' list pending when they are tested on PC.
Thanks & Regards.

**Comment by BlueSky-2024** [comment]
If Vita3K compatible games are crashing on your PC, it could be due to compatibility issues, outdated drivers, or incorrect settings. Make sure your graphics drivers are up to date and check if there are any specific settings needed for Vita3K. For additional troubleshooting tips and resources, you might find useful information at https://hamsterkombat-daily.com/.

**Comment by Olu196** [comment]
Thank you. But I have already clarified that this isn't an issue from my
end. Since my system already plays other 3d games like One Piece 3,
Shinobido 2, Samurai Warriors Chronicles 3, etc at even 3x resolution full
speed. Rather, the issue is that MK:KE is only playable on android & that
is not reflected on your compatibility list.
